app/views/admin/services/_confirm_delete_service.html.haml

```diff
@@ -15,4 +15,4 @@
           #{service_name || service_id}.
     .modal-footer
       %button.btn.btn-default{ 'data-dismiss' => 'modal' }  Close
-      = link_to t('admin.buttons.confirm_delete_service'), { action: :destroy, id: service_id, name: service_name }, method: :delete, class: 'btn btn-danger'
+      = link_to t('admin.buttons.confirm_delete_service'), admin_location_service_path(location_id: location_id, id: service_id), { action: :destroy, class: 'btn btn-danger',  method: :delete }
```

Comment:
Do we need both `action: :destroy` *and* `method: :delete`?

Reply:
yes 😄not sure why!